students/kosarzewski_maciej/lesson_05_lists/greather_than_neighbours.py

```diff
@@ -0,0 +1,7 @@
+input_data = input().split(' ')
+counter = 0
+for x in range(1, len(input_data) - 1):
+    if input_data[x - 1] < input_data[x] \
+            and input_data[x] > input_data[x + 1]:
```

Comment:
It might be shortened to:
if input_data[x - 1] < input_data[x] > input_data[x + 1]:

Anyway, the current implementation is also correct